core/templates/domain/entity_editor_browser_tabs_info/entity-editor-browser-tabs-info.model.ts

```diff
@@ -26,11 +34,45 @@ export interface EntityEditorBrowserTabsInfoObject {
 }
 
 export class EntityEditorBrowserTabsInfo {
-  // Id of the entity whose editor page is opened.
+  /**
+    * It stores the type of the entity for a particular opened
```

Comment:
For all these: drop the initial "It stores". So: "The type ...", "The ID", etc.

Reply:
Done